airbyte-integrations/connectors/destination-snowflake/Dockerfile

```diff
@@ -7,9 +7,28 @@ FROM airbyte/integration-base-java:dev
 #  unzip /tmp/YourKit-JavaProfiler-2021.3-docker.zip -d /usr/local && \
 #  rm /tmp/YourKit-JavaProfiler-2021.3-docker.zip
 
+# amazon linux 2 isn't compatible with urllib3 2.x, so force 1.26.15
+RUN yum install -y python3 python3-devel jq sshpass git gcc-c++ && \
+  alternatives --install /usr/bin/python python /usr/bin/python3 60 && \
+  python -m ensurepip --upgrade && \
+  pip3 install dbt-snowflake==1.0.0 "urllib3<2"
+
+# Luckily, none of normalization's files conflict with destination-bigquery's files :)
+# We don't enforce that in any way, but hopefully we're only living in this state for a short time.
+COPY --from=airbyte/normalization:dev /airbyte /airbyte
```

Comment:
this might actually need to be airbyte/normalization-snowflake. Unclear what the actual difference is, but I just noticed/remembered that those separate images exist.

(similar change would be needed for the airbyte-ci stuff, if true)

Reply:
Documenting your comments re: the airbyte-ci piece

> airbyte-ci is already doing the right thing (i.e. building from airbyte/normalization-snowflake)
> I just built dest-redshift from my branch using airbyte-ci + verified that dbt_project.yml was correct